src/zero/module/aou.clj

```diff
@@ -1,23 +1,400 @@
+  "The sample specific inputs for arrays."
+  {:chip_well_barcode       "7991775143_R01C01"
+   :sample_alias            "NA12878"
+   :red_idat_cloud_path     "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Red.idat"
+   :green_idat_cloud_path   "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Grn.idat"
+   :analysis_version_number 1
+   :sample_lsid             "broadinstitute.org:bsp.dev.sample:NOTREAL.NA12878"
+   :reported_gender         "Female"
+   :params_file             "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/inputs/7991775143_R01C01/params.txt"})
+
+(def chip-metadata
+  "Chip Metadata inputs for arrays."
+  {:bead_pool_manifest_file     "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.bpm"
+   :extended_chip_manifest_file "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.1.3.extended.csv"
+   :cluster_file                "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_CEPH_A.egt"
+   :gender_cluster_file         "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_gender.egt"
+   :zcall_thresholds_file       "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/IBDPRISM_EX.egt.thresholds.txt"}
+  )
+
+(def fingerprinting
+  "Fingerprinting inputs for arrays."
+  {:fingerprint_genotypes_vcf_file       nil
+   :fingerprint_genotypes_vcf_index_file nil
+   :haplotype_database_file              "gs://gcp-public-data--broad-references/hg19/v0/Homo_sapiens_assembly19.haplotype_database.txt"
+   :variant_rsids_file                   "gs://broad-references-private/hg19/v0/Homo_sapiens_assembly19.haplotype_database.snps.list"}
+  )
+
+(def genotype-concordance
+  "Genotype Concordance inputs for arrays."
+  {:control_sample_vcf_file       "gs://broad-gotc-test-storage/arrays/controldata/NA12878.vcf.gz"
+   :control_sample_vcf_index_file "gs://broad-gotc-test-storage/arrays/controldata/NA12878.vcf.gz.tbi"
+   :control_sample_intervals_file "gs://broad-gotc-test-storage/arrays/controldata/NA12878.interval_list"
+   :control_sample_name           "NA12878"})
+
+(def other-inputs
+  "Miscellaneous inputs for arrays."
+  {:call_rate_threshold              0.98
+   :genotype_concordance_threshold   0.98
+   :contamination_controls_vcf       nil
+   :subsampled_metrics_interval_list nil
+   :disk_size                        100
+   :preemptible_tries                3})
+
+(defn- map-aou-environment
+  "Map AOU-ENV to envrionment for inputs preparation."
+  [aou-env]
+  ({:aou-dev "dev" :aou-prod "prod"} aou-env))
+
+(defn array-inputs
+  "Array inputs for ENVIRONMENT that do not depend on the input file."
+  [environment]
+  {:vault_token_path (get-in env/stuff [environment :vault_token_path])
+   :environment      (map-aou-environment environment)})
+
+(comment
+  (array-inputs :aou-dev))
```

Comment:
I don't really have an issue with leaving these `comment`s around to help in future debugging, but it would super helpful if they were briefly commented with an explanation. I also know there was some talk about removing comments when development on this was done--I don't care either way